framework/src/media/audio/audio_manager.c

```diff
@@ -621,6 +621,13 @@ audio_manager_result_t reset_audio_stream_in(void)
 	pcm_close(pcm);
 	pcm = NULL;
 
+	audvdbg("Clean all of an input audio card %d\n", g_actual_audio_in_card_id);
+	g_audio_in_cards[g_actual_audio_in_card_id].status = AUDIO_CARD_READY;
+	g_audio_in_cards[g_actual_audio_in_card_id].device_id = -1;
+	g_audio_in_cards[g_actual_audio_in_card_id].config.volume = 0;
+	g_audio_in_cards[g_actual_audio_in_card_id].pcm = NULL;
```

Comment:
please check line number 621 & 622

Reply:
Jaesik told me that those lines are okay now..